daliuge-translator/dlg/dropmake/lg.graph.schema

```diff
@@ -314,6 +314,11 @@
                                 "keyAttribute": {
                                     "type": "boolean"
                                 },
+                                "encoding": {
+                                    "type": "string",
+                                    "enum": ["binary", "pickle", "dill", "npy", "base64", "utf-8"],
+                                    "default": "utf-8"
```

Comment:
I'm assuming this default is a decision that's been made previously, as opposed to being arbitrary?

Reply:
I agree in principle not to use utf-8. We did not have the ability to specify this per field, but only globally. The previous global default value was pickle, but I think we should switch to dill instead, since that can serialize a lot more than pickle. Also note that this only applies (like before) if a field is used as a port.

Reply:
OK, I'll remove "utf-8" from the options, and set the default to "dill".